webapp/channels/src/components/settings_sidebar/settings_sidebar.tsx

```diff
@@ -110,17 +110,31 @@ export default class SettingsSidebar extends React.PureComponent<Props> {
             pluginTabList = (
                 <>
                     <hr/>
-                    <li
-                        key={'plugin preferences heading'}
-                        role='heading'
-                        className={'header'}
+                    <div
+                        aria-labelledby={'userSettingsModal.pluginPreferences.header'}
+                        role='region'
```

Comment:
I think this should use the group role since regions are bigger areas of the page like the sidebar, centre channel, or RHS

```suggestion
                        role='group'
```